apps/web/src/ui/map/AllSensorsMap.tsx

```diff
@@ -1,6 +1,34 @@
-import useAllSensorsMap from "@/hooks/map/useAllSensorsMap";
 import LoadingSpinner from "../LoadingSpinner";
 import { Sensor } from "@acme/db";
+import isBrowser from "@/utils/isBrowser";
+import dynamic from "next/dynamic";
+import useGeoLocation from "@/hooks/useGeolocation";
+import RotateSpinner from "../RotateSpinner";
+import { percentToSeverity } from "@/utils/percentToSeverity";
+import SensorMarkerPopover from "./SensorMarkerPopover";
+import { Types } from "@acme/leaflet";
+
+const Map = dynamic(
+  () => import("@acme/leaflet").then((mod) => mod.Components.Map),
+  {
+    ssr: false,
+    loading: () => <LoadingSpinner />,
+  },
+);
+
+const Marker = dynamic(
+  () => import("@acme/leaflet").then((mod) => mod.Components.Marker),
+  {
+    ssr: false,
+  },
+);
+
+const Popup = dynamic(
+  () => import("@acme/leaflet").then((mod) => mod.Components.Popup),
+  {
+    ssr: false,
+  },
+);
```

Comment:
These must be imported client-side, so we set `ssr` false. This is because Leaflet is really created from client-side React apps, e.g. using Create React App / Vite, not Next

Reply:
But this workaround works! 🦑